Updates/0171_engineering_membership_cards_duration.sql

```diff
@@ -0,0 +1,2 @@
+UPDATE `item_template` SET `Duration`=1209600, `ExtraFlags`=2 WHERE `entry` IN (10790, 10791);
+
```

Comment:
real tpme ->`ExtraFlags`=**1**

Reply:
You're right, thanks! The wiki is wrong then, [it says 2](https://github.com/cmangos/issues/wiki/Item_template#ExtraFlags).

Reply:
Did we mess up data by this change https://github.com/cmangos/issues/wiki/Item_template/_compare/b63a6f8b1b3f0c1805cc2e7a8dde697a66423b86

aka items having flag 2 when they should have flag 1